x/ccv/provider/keeper/misbehaviour.go

```diff
@@ -109,7 +107,6 @@ func (k Keeper) GetByzantineValidators(ctx sdk.Context, misbehaviour ibctmtypes.
 		}
 	}
 
-	sort.Sort(tmtypes.ValidatorsByVotingPower(validators))
```

Comment:
Sorting not needed because iterating `lightBlock2.Commit.Signatures` always results in the same ordering of validators or ordering does not matter?

Were you trying to avoid non-deterministic validator ordering initially?

Reply:
Initially bc CometBFT is doing it when returning the Byzantine validators, see [GetByzantineValidators](https://github.com/cometbft/cometbft/blob/eba4276bebd7911b5eeb61f27678d6134e58e4f3/types/evidence.go#L253).  However the doc isn't explicit about the requirement, so I dropped it.